### PR TITLE
Propagating Stack-level tags to Athena resources

### PR DESCRIPTION
**Pull request by sskomma**
*Issue #, if available:*
[cloudformation-coverage-roadmap/issues](https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues/1417) 

*Description of changes:*

This change propagates cloud formation stack-level tags to Athena DataCatalog and Workgroup. 
Note: NamedQuery, PreparedStatement do not support tags. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datacatalog/src/main/java/software/amazon/athena/datacatalog/Translator.java [hunk 1]
```diff
@@ -1,7 +1,10 @@
 package software.amazon.athena.datacatalog;
 
-import java.util.ArrayList;
-import java.util.List;
+import java.util.*;
```
Comment (sumeethkc) [inline datacatalog/src/main/java/software/amazon/athena/datacatalog/Translator.java:3]: Please don't use wildcards for imports

File: datacatalog/src/main/java/software/amazon/athena/datacatalog/UpdateHandler.java [hunk 2]
```diff
@@ -1,27 +1,18 @@
...
-
 import com.google.common.collect.Sets;
-import java.util.HashSet;
-import java.util.Set;
-import java.util.stream.Collectors;
 import lombok.Setter;
 import software.amazon.awssdk.services.athena.AthenaClient;
-import software.amazon.awssdk.services.athena.model.AthenaException;
+import software.amazon.awssdk.services.athena.model.*;
+import software.amazon.cloudformation.proxy.*;
```
Comment (sumeethkc) [inline datacatalog/src/main/java/software/amazon/athena/datacatalog/UpdateHandler.java:7]: Same here

File: datacatalog/src/main/java/software/amazon/athena/datacatalog/Translator.java (hunk 1 above)
Comment (sskomma) [reply datacatalog/src/main/java/software/amazon/athena/datacatalog/Translator.java]: fixed ✅

File: datacatalog/src/main/java/software/amazon/athena/datacatalog/UpdateHandler.java (hunk 2 above)
Comment (sskomma) [reply datacatalog/src/main/java/software/amazon/athena/datacatalog/UpdateHandler.java]: fixed ✅

**Review by sumeethkc — APPROVED** [review]

**Pull request merged**